database/migrations/2023_10_16_121201_create_user_details.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('user_details', function (Blueprint $table) {
+            $table->id();
+            $table->integer("user_id")->nullable();;
```

Comment:
User ID cannot be null and must have a unique value. It will also serve as a foreign key referencing the user table.